TheraLang.DAL/Entities/Project.cs

```diff
@@ -3,10 +3,8 @@
 
 namespace TheraLang.DAL.Entities
 {
-    public class Project
+    public class Project : BaseEntity
```

Comment:
This will require to redesign User-Project relation in db

Reply:
You are right. Thats why i asked in the group chat what i have to do. Guys told me that we are going to change models in way, that all of them will be derived of BaseEntity class. So, here we are

Reply:
So now we have link to project creator in 2 places.

Reply:
I think it's okay to keep the user who created the project in the createdby field in the projects table. So what do you suggest? Where is other link ?